test/e2e/helpers/smtpEmails.ts

```diff
@@ -0,0 +1,182 @@
+import { expect } from '@playwright/test'
```

Comment:
same for this helper, would be nice to keep all in one place e.g. `tests/helpers/` folder